asm/nonmatchings/code_317020/world_goombario_pre_battle.s

```diff
@@ -1,7 +1,7 @@
 .set noat      # allow manual use of $at
 .set noreorder # don't insert nops after branches
 
-glabel func_802BDBBC
+glabel world_goombario_pre_battle
```

Comment:
symbol_addrs - I think this is missing?

Reply:
oh maybe we don't want the world funcs there? not sure..

Reply:
I'm not certain on what symbol_addrs.txt is for. It's probably relevant that world partner files are overlays

Reply:
yee nm